saleor/static/dashboard-next/navigation/views/MenuDetails/index.tsx

```diff
@@ -360,8 +361,8 @@ const MenuDetails: React.FC<MenuDetailsProps> = ({ id, params }) => {
                                             menuItem
                                           )}
                                           loading={
-                                            categorySearch.searchOpts.loading ||
-                                            collectionSearch.searchOpts.loading
+                                            categorySearch.result.loading ||
+                                            collectionSearch.result.loading
```

Comment:
it's duplicated, maybe it could be assigned to isLoading variable ?
`const isLoading = categorySearch.result.loading ||
                                            collectionSearch.result.loading`